projects/scion/microfrontend-platform/src/lib/host/manifest-registry/ɵmanifest-registry.ts

```diff
@@ -72,8 +72,8 @@ export class ɵManifestRegistry implements ManifestRegistry, PreDestroy {
     const filter: ManifestObjectFilter = {appSymbolicName, type: intent.type, qualifier: intent.qualifier || {}};
```

Comment:
Why are we passing the qualifier in the filter? I am asking because below, we are passing a qualifier predicate.
Or better, I think, here, we do not need to pass a qualifier predicate at all, simply pass the qualifier in the filter.

Reply:
done